File: 4st/Jisoo/programmer/조건에 맞게 수열 변환하기 3.java
class Solution {
    public int[] solution(int[] arr, int k) {
        int[] result = new int[arr.length];

        if (k % 2 == 1) { 
            for (int i = 0; i < arr.length; i++) {
                result[i] = arr[i] * k;
            }
        } else { 
            for (int i = 0; i < arr.length; i++) {
                result[i] = arr[i] + k;
            }
        }

        return result;
    }
}

File: 4st/Seohyun/programmer/조건에 맞게 수열 변환하기 3.js
function solution(arr, k) {
  var answer;

  if (k % 2 !== 0) {
    answer = arr.map((num) => num * k);
  } else if (k % 2 === 0) {
    answer = arr.map((num) => num + k);
  }

  return answer;
}
